src/system/Time/index.js

```diff
@@ -56,7 +56,7 @@ const Time = ( { time, relativeTime = false, timeOnly = false, className = null,
 };
 
 Time.propTypes = {
-	time: PropTypes.oneOfType( [ PropTypes.string, PropTypes.date ] ),
+	time: PropTypes.oneOfType( [ PropTypes.string, PropTypes.instanceOf( Date ) ] ),
```

Comment:
@brunobasto didn't see it mentioned in the PR, is it a leftover or not? (I believe the new code is correct so I don't consider this an issue, just wondering)

Reply:
Oh, forgot to mention on the PR description, but mentioned on the commit message:

(https://github.com/Automattic/vip-design-system/pull/74/commits/5603609fc5280f83cdda433183171daf6fae5689) 
```
Fix invalid PropType error:

`Warning: Invalid argument supplied to oneOfType. Expected an array of check functions, but received undefined at index 1.`
```

Noticed this error when running tests.